src/System.Net.Primitives/ref/System.Net.Primitives.cs

```diff
@@ -266,6 +266,18 @@ public abstract partial class TransportContext
         protected TransportContext() { }
         public abstract System.Security.Authentication.ExtendedProtection.ChannelBinding GetChannelBinding(System.Security.Authentication.ExtendedProtection.ChannelBindingKind kind);
     }
+
+    public static class HttpVersion
```

Comment:
FYI, it was changed to static in .NET Standard. https://github.com/dotnet/standard/commit/45bbfb6a821b7818b090d38dd65e01d337f9e135